CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 # UNRELEASED
 
+# 0.11.2
+
+## DFX
+
+### fix: disable asset canister redirection of all HTTP traffic from `.raw.ic0.app` to `.ic0.app`
+
+### fix: disable asset canister's ETag HTTP headers
+
+The feature is not yet implemented on `icx-proxy`-level, and is causing 500 HTTP response for some type of assets every second request.
+
+# 0.11.1
+
```

Comment:
This should go after the "unreleased" section, since the "unreleased" section will become 0.12.0, not 0.11.1